src/api/controllers/articleController.js

```diff
@@ -0,0 +1,63 @@
+import Sequelize from 'sequelize';
+
+import models from '../models/index';
+import status from '../../helpers/constants/status.codes';
+import errors from '../../helpers/constants/error.messages';
+
+const userAttrib = ['username', 'profileImage', 'email'];
+const articleAttrib = ['title', 'description', 'slug', 'tagList', 'body'];
+
+const { User, Article } = models;
+/**
+ * containing all article's model controllers (search)
+ *
+ * @export
+ * @class ArticleController
+ */
+export default class ArticleController {
+  /**
+   * search for artcile according to user's request
+   *
+   * @static
+   * @param {*} req the request
+   * @param {*} res the response to be sent
+   * @memberof ArticleController
+   * @returns {Object} res
+   */
+  static async search(req, res) {
+    try {
+      const {
+        title, body, tag
+      } = req.query;
+      const articles = await User.findAll({
```

Comment:
@alainburindi why do `User.findAll` and not `Article.findAll`?

Reply:
because a user has many articles, so when I include if I start with article it's not working

Reply:
for now, the result is the same and if I try to figure out why starting with article is not working it will take much time